pom.xml

```diff
@@ -51,6 +51,8 @@
 		<maven.compiler.source>1.8</maven.compiler.source>
 		<maven.compiler.target>1.8</maven.compiler.target>
 		<version.microprofile.concurrency>1.0-SNAPSHOT</version.microprofile.concurrency>
+		<version.microprofile.config>1.3</version.microprofile.config>
```

Comment:
Not sure if we shouldn't depend on some parent that defined version based on MP release?